context/copy.go

```diff
@@ -93,7 +93,9 @@ func (ctx *Context) CopyPackage(destPath, srcPath, lookRoot, pkgPath string, ign
 	}
 	if h != nil {
 		// Write relative path to GOPATH.
-		h.Write([]byte(strings.Trim(pkgPath, "/")))
+		if _, err := h.Write([]byte(strings.Trim(pkgPath, "/"))); err != nil {
```

Comment:
this looks unnecessary https://godoc.org/hash#Hash
```
// Write (via the embedded io.Writer interface) adds more data to the running hash.
// It never returns an error.
```

Reply:
Fixed.